common/spacing-borders/spacing-borders-en.html

```diff
@@ -0,0 +1,320 @@
+---
+{
+"title": "Spacing and Borders",
+"language":"en",
+"altLangPage":"spacing-borders-fr.html",
+"breadcrumbs":
+[
+{
+"title": "GCWeb home",
+"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html"
+}
+],
+"secondlevel": false,
+"dateModified": "2024-03-05",
+"share": "true"
+}
+---
+<h2><code>.margin-bottom-none</code></h2>
+<div class="row">
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm ">
+		<div class="margin-bottom-none">No bottom Margin</div>
+	</div>
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm ">
+		<div class="margin-bottom-small margin-bottom-none">20 pixel bottom Margin</div>
+	</div>
+	<div class="col-md-4">
+		<pre><code>&lt;div class="margin-bottom-none"&gt;&lt;/div&gt;
+&lt;div class="margin-bottom-small margin-bottom-none"&gt;&lt;/div&gt;</code></pre>
+	</div>
+</div>
+<h2><code>.margin-bottom-small</code></h2>
+<div class="row">
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div class="margin-bottom-small">Small bottom Margin</div>
+	</div>
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div>Default bottom Margin</div>
+	</div>
+	<div class="col-md-4">
+		<pre><code>&lt;div class="margin-bottom-small"&gt;&lt;/div&gt;</code></pre>
+	</div>
+</div>
+<h2><code>.margin-top-large</code></h2>
+<div class="row">
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div class="margin-top-large">Large top Margin</div>
+	</div>
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div>Default top Margin</div>
+	</div>
+	<div class="col-md-4">
+		<pre><code>&lt;div class="margin-top-large"&gt;&lt;/div&gt;</code></pre>
+	</div>
+</div>
+<h2><code>.margin-top-medium</code></h2>
+<div class="row">
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div class="margin-top-medium">Medium top Margin</div>
+	</div>
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div>Default top Margin</div>
+	</div>
+	<div class="col-md-4">
+		<pre><code>&lt;div class="margin-top-medium"&gt;&lt;/div&gt;</code></pre>
+	</div>
+</div>
+<h2><code>.mb-sm-5</code></h2>
+<div class="row">
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div class="mb-sm-5">Bottom Margin of 50</div>
+	</div>
+	<div class="col-md-4 brdr-lft brdr-rght brdr-tp brdr-bttm">
+		<div>No bottom Margin</div>
+	</div>
+	<div class="col-md-4">
+		<pre><code>&lt;div class="mb-sm-5"&gt;&lt;/div&gt;</code></pre>
+	</div>
+</div>
+<div>
+	<h2><code>.mrgn-*-*</code></h2>
```

Comment:
This change is optional for the purpose of this working example but if you do it you will need to update all the other heading to be consistant. And do the same in French.

Ideally we will have a more meaningful name like:

```suggestion
	<h2>Margin generic utilities - <code>.mrgn-*-*</code></h2>
```